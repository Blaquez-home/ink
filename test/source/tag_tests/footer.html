---
layout: content_for
---
{% footer %}
This should go in the footer
{% endfooter %}
