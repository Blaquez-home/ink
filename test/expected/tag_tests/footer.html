
<footer>This should go in the footer</footer>

